backend/src/requests/user-content-request/user-content-request.service.ts

```diff
@@ -50,34 +55,32 @@ export class UserContentRequestService {
    * @throws NotFoundException
    *
    */
-  async getTrendingQuestions(req: any) {
+  async getTrendingQuestions(req: any): Promise<IQuestionMetadata[]> {
     const questions = await this.userContentService.getTrendingQuestions();
     if (null === questions) {
       throw new NotFoundException('No trending questions found.');
     }
-
-    const results: any[] = [];
-    for (let i = 0; i < questions.length; i++) {
-      results.push(
-        await this.getUserContent(
-          questions[i].userContentID,
-          UserContentType.Question,
-          req?.userId,
-        ),
-      );
-    }
-    return results;
+    return await this.mapDatabaseQuestionsToIQuestionMetadata(
+      questions,
+      req?.userId,
+      true,
+    );
   }
 
-  async getQuestionsOfUser(
+  /**
+   * Get Questions of the user in database format
+   * @param userId
+   * @param sortOptions
+   * @private
+   */
+  private async getQuestionsOfUser(
```

Comment:
Is this wrapper method needed?

As the method `myQuestion` could also make the same call of the database service and the sortOptions for that could also be created there.

Reply:
This redundant method was removed within b7a97de